.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com/
```

Comment:
The main reason for specifying this here is to be explicit and account for any `.npmrc` file added by other build systems(corecli) in parent folders.